packages/ui/cypress/tests/watched-accounts.cy.ts

```diff
@@ -186,4 +186,39 @@ describe('Watched Accounts', () => {
     multisigPage.accountHeader().should('be.visible')
     multisigPage.newTransactionButton().should('not.exist')
   })
+
+  it('can not utilize wallet connect when only a watched account', () => {
```

Comment:
This is better maybe?
```suggestion
  it('can not utilize wallet connect with only watched accounts', () => {
```

Reply:
Updated to "can not utilize wallet connect when in watched account mode"